### DISCUSSION TITLE
unsure how to install

### DISCUSSION THREAD

**Discussion by shortcakeslice** [vgmtrans/vgmtrans#225] [Q&A] [answered]
hi ! i downloaded vgmtrans, but i was unsure how im supposed to use it, as i cant see a file that will let me use it. quite confused. sorry if this is very obvious to find !

**Comment by sykhro** [comment, marked as answer]
There isn't an installation procedure. You download the artifact from the Actions tab ([example, down here](https://github.com/vgmtrans/vgmtrans/actions/runs/974071313)), extract the folder wherever and run the program. You'll have to find music files (or games) yourself

**Reply by korterkaks** [reply]
I'm sorry what, why is all this so confusing, how do I download the artifact? I have 0 clue what you said, could I get a better explanation?

**Reply by korterkaks** [reply]
besides, is it just me or you have literally not documented this download process anywhere?

**Reply by sykhro** [reply]
we literally have download links in the readme https://github.com/vgmtrans/vgmtrans/actions/runs/2309171745

**Reply by korterkaks** [reply]
Yeah I couldn't download anything and only now (few sec ago) I found out I had to signed in in order to. I feel betrayed that no one told me about that.

**Reply by sykhro** [reply]
Sadly github nowadays only allows downloads from signed in users. We have recently added downloads to https://vgmtrans.github.io/vgmtrans

**Reply by korterkaks** [reply]
It'd be nice if you had download instructions in the root repository imo.
